### DISCUSSION TITLE
How to know if a custom field is beeing rendered inside the viewPage

### DISCUSSION THREAD

**Discussion by romaldyminaya** [filamentphp/filament#2056] [Help] [answered]
Hi,

I'm trying to create a custom form field, and I need to show some html stuff whether that field is being used in the edit-page or view-page.

Is there any method withing the **Filament\Forms\Components\Field** class to do this ?

**Comment by danharrin** [comment, marked as answer]
Check if the field `$isDisabled()` in the view maybe?

**Reply by danharrin** [reply]
or just check `$this instanceof ViewRecord`

**Reply by romaldyminaya** [reply]
Thanks @danharrin,

The **$isDisabled()** did the Job :)
